content/en/troubleshooting/lambda.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Troubleshoot AWS Lambda Deployment
+linkTitle: AWS Lambda
+weight: 1
+description: >
+  Solutions for issues you might encounter while deploying AWS Lambda functions using the Armory CD-as-a-Service.
+---
+
+
+
+## InvalidParameterValueException
+
+```yaml
+AWSCreateLambdaError: operation error 
+Lambda: CreateFunction, https response error 
+StatusCode: 400, 
+RequestID: ce98341e-8aa6-4c42-99e1-544245303295, 
+InvalidParameterValueException: Error occurred while GetObject. 
+S3 Error Code: PermanentRedirect. 
+S3 Error Message: The bucket is in this region: us-east-2. Please use this region to retry the request
+```
+
+Duplicate functionName in deploy config file
+
+You need to have an S3 bucket with your function archive in the region you want to deploy your function to.
+
+For example (same AWS Account):
+
+| Function        | Target and Region | S3 Bucket Region | Bucket Name   | Function Path                      | 
+|-----------------|-------------------|------------------|---------------|------------------------------------|
+| hello-world.zip | Dev: us-east-1       | us-east-1        | lambda-us-east-1 | s3://lambda-us-east-1/hello-world.zip | 
+| hello-world.zip | Staging: us-east-2  | us-east-2          | lambda-us-east-1 | s3://lambda-us-east-2/hello-world.zip | 
+| hello-world.zip | Infosec: us-west-1   | us-west-1           | lambda-us-east-1 | s3://lambda-us-west-1/hello-world.zip |
+| hello-world.zip | Prod: us-west-2      | us-west-2           | lambda-us-east-1 | s3://lambda-us-west-2/hello-world.zip |
+| hello-world.zip | Prod: eu-central-1      | eu-central-1            | lambda-eu-central-1  | s3://lambda-eu-central-1/hello-world.zip |
+
```

Comment:
can we add a more info link that goes to https://docs.aws.amazon.com/lambda/latest/dg/troubleshooting-deployment.html#troubleshooting-deployment-PermanentRedirect